### PR TITLE
Format modification time according to user timezone

### PR DESCRIPTION
**Pull request by KochTobi**
This PR tries to query the client details for displaying the date in the zone of the client. It falls back to UTC

### CODE REVIEW

File: vaadinfrontend/src/main/java/life/qbic/datamanager/ClientDetailsProvideImpl.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import com.vaadin.flow.component.page.ExtendedClientDetails;
+import com.vaadin.flow.component.page.Page.ExtendedClientDetailsReceiver;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import java.util.Optional;
+
+/**
+ * Receives client details from vaadin and provides them internally.
+ */
+@SpringComponent
+public class ClientDetailsProvideImpl implements ExtendedClientDetailsReceiver,
```
Comment (wow-such-code) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/ClientDetailsProvideImpl.java:12]: Should be named "Provider" instead of "Provide", no?

**Comment by wow-such-code** [comment]
Please provide a description for this PR, the changes seem more complex than the title suggests.

File: vaadinfrontend/src/main/java/life/qbic/datamanager/ClientDetailsProvideImpl.java (hunk 1 above)
Comment (KochTobi) [reply vaadinfrontend/src/main/java/life/qbic/datamanager/ClientDetailsProvideImpl.java:12]: yes

**Review by wow-such-code — APPROVED** [review]
thanks, lgtm!

**Pull request merged**